docs/general/sentence_processing_and_feature_extraction_techniques_for_luigi.md

```diff
@@ -0,0 +1,109 @@
+| Lemmatization                     | `nlp.prop_lemmas`             | Assigning the base forms of words                           | 'Revolutions' has the lemma: 'revolution' <br>  'was' would have the lemma: 'be'|
+| Named Entity Recognition (NER)    | `nlp.prop_ner`                | Labelling "real-world" objects ("NE"=Named Entity)          | 'six' has the NE: 'CARDINAL', which are numerals that do not fall under another type <br> [List](https://spacy.io/api/annotation#named-entities) of NEs|
+| Triple Extraction                 | `nlp.prop_triples`            | A triple consists of subject, predicate, object of sentence | Triple: subject: 'Revolutions', predicate: 'need', object: 'chickens' |
+| About Roboy                       | `nlp.prop_roboy`              | Detecting whether sentence is about Roboy                   | 'you', 'roboy', 'robot', 'roboboy', ... |
+| Yes-No                            | `nlp.prop_yesno`              | Detecting answers to yes-no questions                       | Checking for 'yes', 'no', 'i don't know', 'probably', 'probably not' and synonyms of these                           |
+| Sentence Type: Question           | `nlp.sig_is_question`         | Emitted if the input sentence is a question                 |                                                |
+| Play Game                         | `nlp.sig_intent_play`         | Emitted when the interlocutor wants to play a game          | input: "I want to play", "I like games" or something similar    |
+
+### Using the Features
+
+#### Detailed POS Tagging
+
+This feature can be used to understand the requested flavors or payment options. 
+Roboy's flavor and payment options are predefined so any match between our database and incoming token
+can help us to identify the requested options. Such as:
+
+- vanilla
+- chocolate
+- lemon
+- strawberry
+
+or 
+
+- credit card
+- cash
+- bitcoin
+
+Alternatively by using POS, Roboy can understand whether the sentence is a request or not by only parsing the verb.
+For example if the customer uses one of the following phrases, it is easy to detect that customer has a request:
+
+- would like
+- can have
+- can get
+- will have
+- choose
+- wish
+- desire
+- prefer
+- could do with
+- have an urge for
+
+#### Yes-No Property
+
+This feature can be used to understand a given question's positive/negative answer to proceed such as:
+
+- Do you want the usual?
+- Do you want ice cream?
+
+Example: State that reads the "yesno" property
+
+```python
+import ravestate as rs
+import ravestate_nlp as nlp
+import ravestate_rawio as rawio
+
+@rs.state(
+    cond=nlp.prop_yesno.changed(),  # state reacts to a change in the 'yesno' property
```

Comment:
Maybe add a state that emit's some `question-asked` signal, to increase the value of this example: This state reacts when the user says yes or no, in response to some question asked before.